game-core/src/test/java/games/strategy/util/VersionTest.java

```diff
@@ -80,28 +77,23 @@ public void testIsLessThan() {
     assertFalse(new Version(2, 0).isLessThan(new Version(1, 0)));
   }
 
-  @Test
-  public void testWithMicro() {
```

Comment:
Instead of removing this test, it should probably be replaced with an equivalent test for the new `withPoint()` method.